functions.php

```diff
@@ -0,0 +1,42 @@
+<?php
+function dbpull():array{
```

Comment:
should be `dbPull()` - camelcase everywhere

Reply:
also better to rename to `pullAllAnimalsFromDb()` or something like that - makes it a bit easier to tell exactly what it does